core/src/test/scala/org/apache/spark/status/protobuf/KVStoreProtobufSerializerSuite.scala

```diff
@@ -773,4 +740,568 @@ class KVStoreProtobufSerializerSuite extends SparkFunSuite {
+    assert(result.info.shuffleRemoteBytesReadToDisk == input.info.shuffleRemoteBytesReadToDisk)
+    assert(result.info.shuffleLocalBytesRead == input.info.shuffleLocalBytesRead)
+    assert(result.info.shuffleReadBytes == input.info.shuffleReadBytes)
+    assert(result.info.shuffleReadRecords == input.info.shuffleReadRecords)
+    assert(result.info.shuffleWriteBytes == input.info.shuffleWriteBytes)
+    assert(result.info.shuffleWriteTime == input.info.shuffleWriteTime)
+    assert(result.info.shuffleWriteRecords == input.info.shuffleWriteRecords)
+
+    assert(result.info.name == input.info.name)
+    assert(result.info.description == input.info.description)
+    assert(result.info.details == input.info.details)
+    assert(result.info.schedulingPool == input.info.schedulingPool)
+
+    assert(result.info.rddIds == input.info.rddIds)
+    assert(result.info.accumulatorUpdates, input.info.accumulatorUpdates)
+
+    assert(result.info.tasks.isDefined == input.info.tasks.isDefined)
+    if (result.info.tasks.isDefined && input.info.tasks.isDefined) {
+      assertIdTask(result.info.tasks.get, input.info.tasks.get)
+    }
+
+    assert(result.info.executorSummary.isDefined == input.info.executorSummary.isDefined)
+    if (result.info.executorSummary.isDefined && input.info.executorSummary.isDefined) {
+      assert(result.info.executorSummary.get, input.info.executorSummary.get)
+    }
+
+    assert(result.info.speculationSummary.isDefined == input.info.speculationSummary.isDefined)
+    if (result.info.speculationSummary.isDefined && input.info.speculationSummary.isDefined) {
+      assert(result.info.speculationSummary.get, input.info.speculationSummary.get)
+    }
+    assert(result.info.killedTasksSummary == input.info.killedTasksSummary)
+    assert(result.info.resourceProfileId == input.info.resourceProfileId)
+    assert(result.info.peakExecutorMetrics.isDefined == input.info.peakExecutorMetrics.isDefined)
+    if (result.info.peakExecutorMetrics.isDefined && input.info.peakExecutorMetrics.isDefined) {
+      assert(result.info.peakExecutorMetrics.get, input.info.peakExecutorMetrics.get)
+    }
+    assert(result.info.taskMetricsDistributions.isDefined ==
+      input.info.taskMetricsDistributions.isDefined)
+    if (result.info.taskMetricsDistributions.isDefined &&
+      input.info.taskMetricsDistributions.isDefined) {
+      assert(result.info.taskMetricsDistributions.get, input.info.taskMetricsDistributions.get)
+    }
+    assert(result.info.executorMetricsDistributions.isDefined ==
+      input.info.executorMetricsDistributions.isDefined)
+    if (result.info.executorMetricsDistributions.isDefined &&
+      input.info.executorMetricsDistributions.isDefined) {
+      assert(result.info.executorMetricsDistributions.get,
+        input.info.executorMetricsDistributions.get)
+    }
+  }
+
+  private def assert(result: TaskMetrics, input: TaskMetrics): Unit = {
```

Comment:
nit: rename all the `assert` methods as `checkAnwser(result, expected)`?

Reply:
Done